test/categorical_algebra/Diagrams.jl

```diff
@@ -71,6 +71,18 @@ d = dom(f)
 @test codom(op(f)) == op(dom(f))
 @test op(g)⋅op(f) == op(f⋅g)
 
+# param_compose, should probably be in FinCats
```

Comment:
Remove since not part of public interface and implicitly tested by data migration pipeline.